dashboard/config/linux/bits/subsystems.yml

```diff
@@ -442,6 +447,12 @@ config:
  - CHROME_PLATFORMS: n
  - AMD_IOMMU: n
  - INTEL_IOMMU: [x86_64]
+ - IOMMUFD: [linux-next]
+ - IOMMUFD_TEST: [linux-next]
+ # Generally we should not enable this since we don't run any boot unit tests.
+ # The only reason to enable it is IOMMUFD_TEST, which really shouldn't
+ # depend on RUNTIME_TESTING_MENU since it only adds debug checks.
```

Comment:
It does actually add a whole bunch of runtime code built into the module, including an entire uAPI surface to support controlling that code.